### PR TITLE
Peggy smart contract docs

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: smart-contracts/.env.example [hunk 1]
```diff
@@ -2,23 +2,26 @@
...
 LOCAL_PROVIDER="http://localhost:7545"
-# Local validator count must be at least 1 and no more than 8 
 
 # ------------
 #   Network
 # ------------
 # Owner of the bridgebank, this must be set when we are deploying to tesnet or mainnet
 OWNER='0x627306090abaB3A6e1400e9345bC60c78a8BEf57'
+# Address of the pauser
+PAUSER='0x627306090abaB3A6e1400e9345bC60c78a8BEf57'
```
Comment (banshee) [inline smart-contracts/.env.example:19]: Any chance we could change that environment variable name to something more specific like ETHEREUM_PAUSER? Having very generic environment variables like OWNER and PAUSER in setups that have to deal with both blockchains is confusing.

**Review by banshee — APPROVED** [review]

**Review by juniuszhou — APPROVED** [review]

File: smart-contracts/contracts/BridgeBank/BridgeBank.sol [hunk 2]
```diff
@@ -48,33 +52,32 @@ contract BridgeBank is BankStorage,
...
-
-
     /*
      * @dev: Modifier to restrict access to the cosmos bridge
      */
     modifier onlyCosmosBridge() {
         require(
             msg.sender == cosmosBridge,
-            "Access restricted to the cosmos bridge"
+            "!cosmosbridge"
```
Comment (musnit) [inline smart-contracts/contracts/BridgeBank/BridgeBank.sol:80]: why make the error messages harder to read/understand?

File: smart-contracts/contracts/BridgeBank/BridgeBank.sol [hunk 3]
```diff
@@ -83,11 +86,21 @@ contract BridgeBank is BankStorage,
      * @dev: Modifier to only allow valid sif addresses
      */
     modifier validSifAddress(bytes memory _sifAddress) {
-        require(_sifAddress.length == 42, "Invalid sif address length");
-        require(verifySifPrefix(_sifAddress) == true, "Invalid sif address prefix");
+        require(_sifAddress.length == 42, "Invalid len");
```
Comment (musnit) [inline smart-contracts/contracts/BridgeBank/BridgeBank.sol:89]: why make the error messages harder to read/understand?

File: smart-contracts/contracts/BridgeBank/BridgeBank.sol [hunk 4]
```diff
@@ -153,11 +165,11 @@ contract BridgeBank is BankStorage,
         if (_inList) {
             // if we want to add it to the whitelist, make sure that the address
             // is 0, meaning we have not seen that symbol in the whitelist before
-            require(listAddress == address(0), "Token already whitelisted");
+            require(listAddress == address(0), "whitelisted");
```
Comment (musnit) [inline smart-contracts/contracts/BridgeBank/BridgeBank.sol:168]: why make the error messages harder to read/understand?

**Pull request merged**